### PR TITLE
[GH-#260] Added minimum fee

### LINKED ISSUE #260: adjust minimum fee
adjust minimum fee, greater equals a configurable value

(keep the fee calculations code, as I assume this will get changed again in the future)

### PR DESCRIPTION
**Pull request by HristianHristov**
Resolves #260

### CODE REVIEW

File: apps/aecore/lib/aecore/structures/oracle_extend_tx.ex [hunk 1]
```diff
@@ -92,7 +94,7 @@ defmodule Aecore.Structures.OracleExtendTx do
       !Map.has_key?(registered_oracles, sender) ->
         throw({:error, "Account isn't a registered operator"})
 
-      fee < calculate_minimum_fee(tx.ttl) ->
+      fee < calculate_minimum_fee(tx.ttl) && fee < @minimum_fee ->
```
Comment (cytadela8) [inline apps/aecore/lib/aecore/structures/oracle_extend_tx.ex]: There should be `||` here for correct behavior probably.

Comment (thepiwo) [reply apps/aecore/lib/aecore/structures/oracle_extend_tx.ex]: gues this wouldn't be compatible if our fee is lower than epochs

**Comment by d-velev** [comment]
I see that the fee requirement is added to oracle transactions as well, is that the case in epoch? It would make their base oracle fee settings useless

**Comment by HristianHristov** [comment]
@d-velev  So we should remove the minimum_fee from the oracles ?

**Comment by d-velev** [comment]
@HristianHristov yes they have a fee requirement of their own

File: apps/aecore/lib/aecore/structures/spend_tx.ex [hunk 2]
```diff
@@ -147,7 +147,8 @@ defmodule Aecore.Structures.SpendTx do
             Application.get_env(:aecore, :tx_data)[:miner_fee_bytes_per_token]
         end
 
-      tx.data.fee >= Float.floor(tx_size_bytes / bytes_per_token)
+      tx.data.fee >= Float.floor(tx_size_bytes / bytes_per_token) &&
```
Comment (thepiwo) [inline apps/aecore/lib/aecore/structures/spend_tx.ex]: this still uses byte size fee calculation

Comment (thepiwo) [reply apps/aecore/lib/aecore/structures/spend_tx.ex]: @HristianHristov what do you think about this?

Comment (HristianHristov) [reply apps/aecore/lib/aecore/structures/spend_tx.ex]: Yes It seems I forgot about this one, will be removed.

**Comment by thepiwo** [comment]
@cytadela8 can you re-review?

**Pull request merged**